### PR TITLE
change version to match latest npm requirement for x.x.x versioning

### PR DESCRIPTION
**Pull request by prosenberg1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "author": "NuoDB, Inc.",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
   "license": "Apache-2.0",
-  "version": "3.5",
+  "version": "3.5.0",
```
Comment (dwhite-nuodb) [inline package.json:6]: Would prefer this change be marked as 3.5.1

**Review by dwhite-nuodb — CHANGES_REQUESTED** [review]
Make changes to be a change for DBaaS compatibility to be earmarked in a particular distinct way.

File: package.json (hunk 1 above)
Comment (prosenberg1) [reply package.json]: Changed to 3.5.1 👍

**Review by dwhite-nuodb — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (adriansuarez) [reply package.json]: Why would this be 3.5.1 if it is on master? Normally `x.y.0` releases are made on master and `x.y.z` releases are made off of development branches for `x.y`, where `z` is non-0.

It does not make sense to be making patch releases on master. This should be 0.

**Pull request merged**